editoast/src/views/timetable/track_occupancy_utils.rs

```diff
@@ -0,0 +1,269 @@
+use chrono::DateTime;
+use chrono::Duration;
+use chrono::Utc;
+use core_client::simulation::ReportTrain;
+use schemas::infra::TrackOffset;
+use schemas::primitives::PositiveDuration;
+use schemas::train_schedule::OperationalPointIdentifier;
+use schemas::train_schedule::PathItemLocation;
+
+use crate::views::path::path_item_cache::PathItemCache;
+use crate::views::path::pathfinding::PathfindingResult;
+use crate::views::path::projection::PathProjection;
+use crate::views::path::projection::TrackLocationFromPath;
+use crate::views::projection::find_index_upper;
+use crate::views::projection::linear_interpolate;
+use crate::views::timetable::simulation;
+use crate::views::timetable::simulation::SimulationResponseSuccess;
+
+#[derive(Debug, Clone)]
+pub struct TrackOccupancyResult {
```

Comment:
```suggestion
pub(super) struct TrackOccupancy {
```

In most cases, the `-Result`, `-Data`, etc. suffixes are noise IMO.

Reply:
Done.